pkg/imager/imager.go

```diff
@@ -112,7 +112,7 @@ func (i *Imager) Execute(ctx context.Context, outputPath string, report *reporte
 			return "", fmt.Errorf("UKI output is not supported in this Talos version")
 		}
 	case profile.OutKindISO, profile.OutKindImage, profile.OutKindInstaller:
-		needBuildUKI = needBuildUKI && i.prof.SecureBootEnabled()
+		// needBuildUKI = needBuildUKI && i.prof.SecureBootEnabled()
```

Comment:
not sure what to do here? :thinking:

Reply:
I think we need to split it up:

* for ISO & Image things stay same as before
* for Installer we build UKI if SecureBoot || Talos >= 1.10